### PR TITLE
Supervisor view Courses

### PR DESCRIPTION
**Pull request by lethihaivan**
Screenshot from 2021-03-30 13-45-38
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/34993)

## WHAT
- Supervisor can view all course

## Evidence (Screenshot or Video)
Screenshot from 2021-03-30 13-46-53

### CODE REVIEW

**Comment by lethihaivan** [comment]
ready

File: app/helpers/courses_helper.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module CoursesHelper
+  def load_coures_statuses
+    @status = Course.statuses.keys
```
Comment (vanvtt-0952) [inline app/helpers/courses_helper.rb]: `@status` -> số nhìu

File: app/views/supervisor/courses/_course.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <span class="badge badge-warning"><%= course.status %></span>
+        <a class="text-decoration-none text-muted">
+          <%= link_to course.name, supervisor_course_path(course.id)%>
+        </a>
+      </h4>
+    </li>
+    <li class="mb-1">
+      <i class="fa fa-clock"></i>
+        <span>
+          <strong><%= course.start_date %>--></strong>
```
Comment (vanvtt-0952) [inline app/views/supervisor/courses/_course.html.erb]: format thời gian voiw I18n đi em nhé

File: db/seeds.rb [hunk 3]
```diff
@@ -115,29 +136,25 @@
...
+  subject_id = Subject.all.pluck(:id).sample
   Task.create!(name: name,
   subject_id: subject_id)
 end
 
 99.times do |n|
-  course_id = Faker::Number.within(range: 1..6)
-  user_id = Faker::Number.within(range: 1..99)
+  course_id = Course.all.pluck(:id).sample
+  user_id = User.all.pluck(:id).sample
```
Comment (vanvtt-0952) [inline db/seeds.rb]: `User.pluck(:id).sample`
-> ko cần `.all` đâu bé, check bên duwois nhé

**Comment by lethihaivan** [comment]
Screenshot from 2021-03-30 15-31-24
ready

**Comment by lethihaivan** [comment]
ready

**Review by tham-1781 — APPROVED** [review]

**Pull request merged**